docs/userguide/networking/get-started-overlay.md

```diff
@@ -62,15 +62,12 @@ store) key-value stores. This example uses Consul.
 
 	 You passed the `docker run` command the connection configuration using a bash
 	 expansion `$(docker-machine config mh-keystore)`.  The client started a
-	 `progrium/consul` image running in the `mh-keystore` machine. The server is called `consul`and is listening port `8500`.
+	 `progrium/consul` image running in the `mh-keystore` machine. The server is 
+	 called `consul` and it is listening on port `8500`.
 
-4. Set your local environment to the `mh-keystore` machine.
+4. Run the `docker ps` command to see the `consul` container.
 
-			$  eval "$(docker-machine env mh-keystore)"
-
-5. Run the `docker ps` command to see the `consul` container.
```

Comment:
I'll leave removing this step to @moxiegirl (perhaps she wants the switching to this machine as an explicit step)

Was there a technical issue with this step being there?